apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/GridAreas/Client/GridAreasClient.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
+using NodaTime;
+
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.GridAreas.Client;
+
+public class GridAreasClient(IMarketParticipantClient_V1 client) : IGridAreasClient
+{
+    public async Task<GridAreaOverviewItemDto> GetGridAreaOverviewItemByIdAsync(Guid gridAreaId) =>
+        (await client.GridAreaOverviewAsync()).First(x => x.Id == gridAreaId);
+
+    public async Task<IEnumerable<GridAreaOverviewItemDto>> GetGridAreaOverviewItemsAsync() =>
+        await client.GridAreaOverviewAsync();
+
+    public async Task<IEnumerable<GridAreaDto>> GetGridAreasAsync(CancellationToken ct = default)
+    {
+        var actorsTask = client.ActorGetAsync(ct);
+        var gridAreasTask = client.GridAreaGetAsync(ct);
+        var actors = await actorsTask;
+        var gridAreas = await gridAreasTask;
+
+        // TODO: Does this do anything?
+        var consolidations = await client.ActorConsolidationsAsync(ct);
```

Comment:
@mimse Do you know if this can be removed?

Reply:
I think it was added by a mistake in this PR https://github.com/Energinet-DataHub/greenforce-frontend/pull/3832

Reply:
It can be removed

Reply:
Great, I have a follow up PR coming up, will remove it there